src/main/java/com/scmspain/karyon/accesslog/formatters/JsonLog.java

```diff
@@ -15,6 +20,19 @@
 
   @Override
   public String format(AccessLog logLine) {
-    return gson.create().toJson(logLine);
+
+    Map<String, String> accessLogMapped = new HashMap<>();
+    accessLogMapped.put("clientIp", logLine.clientIp());
+    accessLogMapped.put("date",
+        new SimpleDateFormat("dd/MM/yyyy:HH:mm:ss.SSS Z").format(Date.from(logLine.date())));
+    accessLogMapped.put("method", logLine.method());
+    accessLogMapped.put("uri", logLine.uri());
+    accessLogMapped.put("httpVersion", logLine.httpVersion());
+    accessLogMapped.put("referrer", String.valueOf(logLine.referer()));
+    accessLogMapped.put("userAgent", logLine.userAgent());
+    accessLogMapped.put("responseSize", String.valueOf(logLine.responseSize()));
+    accessLogMapped.put("timeTaken", String.valueOf(logLine.timeTaken()));
+
+    return gson.create().toJson(accessLogMapped);
```

Comment:
At some point we'll replace this by using serializer instead, but from now is the best way to have json.